### PR TITLE
CMake Readme Fixes

### PR DESCRIPTION
**Pull request by AchimTuran**
During my first tests with our new CMake based build system I noticed that the instructions for Windows are not entirely correct.

@fetzerch 
This PR includes the fixes we discussed. Can you have a look at it?

@afedchin 
Do you know since when we use fxc.exe for shader compilation? I noticed that this part is missing in the CMake build instructions.
Can you confirm that we have to install the Windows SDK for Windows 7, 8.0, 8.1 or 10 to get our build working? I think this is the best MS link for downloading the SDK's, because it shows all available downloads. Or do you have a better link?

### CODE REVIEW

File: project/cmake/README.md [hunk 1]
```diff
@@ -87,7 +88,7 @@ cmake --build .  # or: nmake
 ```
 cmake -G "Visual Studio 12" <KODI_SRC>/project/cmake/
 cmake --build . --config "Debug"  # or: Build solution with Visual Studio
-KODI_HOME=%CD% Debug/kodi.bin
+set KODI_HOME="%CD%" && Debug\kodi.exe`
 ```
```
Comment (fetzerch) [inline project/cmake/README.md]: that last '`' shouldn't be there I guess

Comment (AchimTuran) [inline project/cmake/README.md]: Thanks, you're right. Copy paste from Slack chat log. I will fix it.

**Comment by afedchin** [comment]
@AchimTuran WinSDK should be installed with VS2013, no need to install it separately

**Comment by afedchin** [comment]
about fxc we use it since migration to dx11. here is fix for it #9135

**Comment by AchimTuran** [comment]
@afedchin
Thanks for the quick response.

>WinSDK should be installed with VS2013, no need to install it separately
I like to keep the links and the information for reference. What do you think about that idea? Should I completly remove the information or should we add that to the wiki?

**Comment by afedchin** [comment]
You can keep information about SDK just as a note 'if something goes wrong during compilation try to re-install Windows SDK bla-bla'.

**Comment by AchimTuran** [comment]
@fetzerch & @afedchin 
From my side this PR is ready. If you agree with me, feel free to merge. Otherwise please mention the show stoppers and I will fix them.

**Comment by afedchin** [comment]
looks good

**Pull request merged**